salt/orchestrate/edx/init.sls

```diff
@@ -93,12 +102,19 @@ build_edx_nodes:
     - require:
         - salt: deploy_consul_agent_to_edx_nodes
 
-stop_non_edx_worker_services:
 {% for service in ['edxapp:', 'forum', 'xqueue', 'xqueue_consumer'] %}
+stop_non_edx_worker_services_{{ service }}:
   salt.function:
     - name: supervisord.dead
     - tgt: 'G@roles:edx-worker and G@environment:{{ ENVIRONMENT }}'
     - tgt_type: compound
-    - bin_env: '/edx/bin/supervisorctl'
-    - name: {{ service }}
+    - kwargs:
+        bin_env: '/edx/bin/supervisorctl'
+        name: {{ service }}
+{% endfor %}
+
+{% for bucket in bucket_prefixes %}
+create_edx_s3_bucket_{{ bucket }}_{{ PURPOSE_PREFIX }}:
+  boto_s3_bucket.create:
```

Comment:
This should use the `.present` state function instead of the `create` function from the execution module.